docs/src/main/asciidoc/kubernetes-client.adoc

```diff
@@ -29,6 +29,39 @@ This will add the following to your pom.xml:
     </dependency>
 ----
 
+== Usage
+
+Quarkus configures a Bean of type `KubernetesClient` which can be injected into application code using the well known CDI methods.
+This client be can be configured using various properties as can be seen in the following example:
```

Comment:
```suggestion
This client can be configured using various properties as can be seen in the following example:
```
I thought I mentioned it the first time but maybe I didn't click on the button.

Reply:
I probably missed it